### PR TITLE
AB#720: updated READMEs

### PR DESCRIPTION
**Pull request by Anthony-Mckale**
updating readmes to be cleanedup and reference riversafe helpdesk

[AB#720](https://dev.azure.com/riversafe/0b0abc1f-f46a-4d96-a470-e1c546e3cf29/_workitems/edit/720)

added fixes for missing semgrep and java tool versions

updated tables to be cyclonedx

### CODE REVIEW

File: README-DOCKER.md [hunk 1]
```diff
@@ -7,102 +7,83 @@ The default eze-cli Dockerfile contains the eze cli, and a selection of common o
...
+ | SCA    | node-npmoutdated     | 8.1.4         | NPM        | NODE                               | opensource node outdated dependency scanner
+ |
+ | SBOM   | node-cyclonedx       | 3.3.1         | Apache 2.0 | NODE                               | opensource node bill of materials (SBOM) generation utility
+ |
+ | SCA    | container-trivy      | 0.18.2        | Apache 2.0 | CONTAINER                          | opensource container scanner
+ |
+ | SCA    | kics                 | 1.4.8         | Apache 2.0 | CONTAINER                          | opensource infrastructure scanner
+ |
+
 ```
```
Comment (michaelobineche-rs) [inline README-DOCKER.md:80]: image

image

possible formatting error

Comment (Anthony-Mckale) [reply README-DOCKER.md:80]: lol it's just your screen isn't big enough :)

it soft wraps around

Comment (Anthony-Mckale) [reply README-DOCKER.md:80]: was thinking of rewriting it to output markdown table as well as the console table

that way i could copy and paste easier

Comment (Anthony-Mckale) [reply README-DOCKER.md:80]: actually ignore me you're right it is a formatting issue, fixing :)

Comment (Anthony-Mckale) [reply README-DOCKER.md]: fixed
image

**Review by mgonzales-rs — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -121,6 +121,64 @@ Configuration Format for SemGrep
...
+| bom-formatted | 0.15.2       | Apache-2.0 | bill of materials multiformat reporter |
+| sarif         | 0.12.0-alpha | inbuilt    | sarif output file reporter             |
+
+_Updated: 18/01/2022_
+
+An updated list of reporters can be found using the command
+
+```bash
+docker run -t --rm riversafe/eze-cli reporters list --include-source-type
+docker run -t --rm riversafe/eze-cli reporters help <report-name>
```
[the remark this answers is not in the document]
Comment (Anthony-Mckale) [reply README.md:178]: removed the tools list from the docker dpcs

[the remark this answers is not in the document]
Comment (Anthony-Mckale) [reply README.md]: > could we change it to "reporter-name" instead of "report-name"?

updated

**Pull request merged**